update/release_image.go

```diff
@@ -45,18 +45,18 @@ func ParseReleaseKind(s string) (ReleaseKind, error) {
 const UserAutomated = "<automated>"
 
 type ReleaseContext interface {
-	SelectServices(Result, []ControllerFilter, []ControllerFilter) ([]*ControllerUpdate, error)
+	SelectWorkloads(Result, []WorkloadFilter, []WorkloadFilter) ([]*WorkloadUpdate, error)
 	Registry() registry.Registry
 }
 
 // NB: these get sent from fluxctl, so we have to maintain the json format of
 // this. Eugh.
 type ReleaseImageSpec struct {
-	ServiceSpecs []ResourceSpec
-	ImageSpec    ImageSpec
-	Kind         ReleaseKind
-	Excludes     []flux.ResourceID
-	Force        bool
+	WorkloadSpecs []ResourceSpec `json:"serviceSpecs"`
```

Comment:
Changing the struct tags makes me nervous -- in my experience, unexpected serialisations can make things tricky to track down. Mind you, the reason this is a problem is that this struct is used in the API -- what we should really do is have a versioned type for the API, and convert to the "timeless" internal types here. But perhaps better to leave this one for now.